pubsub/pubsub.go

```diff
@@ -274,26 +274,20 @@ func defaultBatcher(ctx context.Context, t *Topic, dt driver.Topic) driver.Batch
 		}
 		return nil
 	}
-	return batcher.New(reflect.TypeOf(&driver.Message{}), maxHandlers, handler)
+	return batcher.New(reflect.TypeOf(&driver.Message{}), batcherOpts, handler)
 }
 
 // newTopic makes a pubsub.Topic from a driver.Topic.
 //
-// If newBatcher is nil, a default batcher implementation will be used.
-// If non-nil, the driver.Batcher returned by newBatcher should batch
-// *driver.Message, and call the SendBatch func of the driver.Topic in its
-// handler. See defaultBatcher for a sample implementation.
-func newTopic(d driver.Topic, newBatcher func(context.Context, *Topic, driver.Topic) driver.Batcher) *Topic {
+// batcherOpts may be nil to accept defaults.
+func newTopic(d driver.Topic, batcherOpts *batcher.Options) *Topic {
```

Comment:
ditto. We can see the type name is `batcher.Options`.

Reply:
Done.